community/basic_examples/hw_aware_synthesis.ipynb

```diff
@@ -0,0 +1,601 @@
+{
```

Comment:
<p>You can write <code>my_mcx</code> instead of my_mcx, then it will look nicer. relevant for all mentions of function and variable names in the markdown parts </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>